src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -48,19 +53,40 @@ export class AssetsTable extends React.Component {
     });
   }
 
-  addDeleteButton() {
-    const newAssetsList = this.props.assetsList.map((asset) => {
+  getNextFocusElementOnDelete(assetToDelete) {
```

Comment:
Done! And agreed, it is clearer.

Reply:
Actually, after reading more of your comments -- it's determining where to focus after closing the StatusAlert that appears on delete. I'm not sure if the `OnDeleteClose` is clear about that, but if we're going to be using the status alert for multiple alert types (delete/copy URL/...) I'm not sure if `OnAlertClose` will be clear either because that will have different rules, though I guess we could maybe extend that function to encompass all the rules related to the alert and focus?

Reply:
Ah, I see! Maybe the current name makes the most sense actually. I'm reading it again and finding it to be clear!